cdhweb/pages/models.py

```diff
@@ -211,9 +211,8 @@ def get_context(self, request):
         '''Add featured updates, projects, and events to the page context.'''
         context = super().get_context(request)
 
-        # FIXME an ImportError is thrown when classes in this module are
-        # imported elsewhere if any of these three models are imported at the
-        # top. Why?
+        # FIXME because these apps import LandingPage, there is a circular
```

Comment:
To me it's a FIXME because it's probably not great to be fetching the models via `apps.get_model()` on every call to `get_context()`, although I don't really know the solution. Maybe it's enough to fetch the models once on the class, so they're available in `get_context()`? I was hoping there was more of a django standard way of dealing with circular imports. Do you have any ideas?

Reply:
I agree it's not ideal. I'm not sure there is an easy solution, though.  I think normally in django you would have views separate from the models and it wouldn't be a problem.

Let's keep it as a fixme for now so we can keep thinking about a better solution. Importing them once would probably be better than importing every time, but it still seems kind of awkward.